client/scss/components/_image_gallery.scss

```diff
@@ -30,3 +32,160 @@
+  }
+}
+
+.image-gallery__slider-inner {
+  overflow: visible;
+  background: color('white');
+  padding: 100px map-get($container-padding, 'small') 0;
+
+  @include respond-to('medium') {
+    padding-left: map-get($container-padding, 'medium');
+    padding-right: map-get($container-padding, 'medium');
+  }
+
+  @include respond-to('large') {
+    padding-left: map-get($container-padding, 'large');
+    padding-right: map-get($container-padding, 'large');
+  }
+}
+
+.image-gallery__slides-container { // these width and left calculations are related to the gallery appearing in the article template
+  margin: 0;
+  position: relative;
+  width: calc(100vw - #{map-get($container-padding, 'small')});
+
+  @include respond-to('medium') {
+    width: calc(100vw - #{map-get($container-padding, 'medium')} - 12.5%);
+    margin-left: 12.5%;
+  }
+
+  @include respond-to('large') {
+    width: calc(100vw - #{map-get($container-padding, 'large')});
+    margin-left: 0;
+  }
+
+  @include respond-to('xlarge') {
+    width: $container-width-max - (map-get($container-padding, 'large') * 2);
+    margin-left: 50%;
+    left: (($container-width-max - (map-get($container-padding, 'large') * 2)) / 2) * (-5/6);
+  }
+}
+
+.image-gallery__slide-item {
+  opacity: 0.2;
+  transition: opacity 0.7s ease;
+  align-self: flex-start;
+
+  .captioned-image__image-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    background: color('smoke');
+    height: 204px;
+
+    @include respond-to('medium') {
+      height: 640px;
+    }
+  }
+}
+
+.image-gallery__slide-item--current {
+  opacity: 1;
+}
+
+.image-gallery__slider-controls {
+  background: color('white');
+  min-height: 40px;
+  padding: $v-spacing-unit * 3 map-get($container-padding, 'small');
+  margin-bottom: $v-spacing-unit * 3;
+  margin-left: map-get($container-padding, 'small') * -1;
+  margin-right: map-get($container-padding, 'small') * -1;
+  text-align: right;
+
+  @include respond-to('medium') {
+    margin-left: map-get($container-padding, 'medium') * -1;
+    margin-right: map-get($container-padding, 'medium') * -1;
+    padding-left: map-get($container-padding, 'medium');
+    padding-right: map-get($container-padding, 'medium');
+    text-align: left;
+  }
+
+  @include respond-to('large') {
+    margin-left: map-get($container-padding, 'large') * -1;
+    margin-right: map-get($container-padding, 'large') * -1;
+    padding-left: map-get($container-padding, 'large');
+    padding-right: map-get($container-padding, 'large');
+  }
+  /* stylelint-disable plugin/selector-bem-pattern */
+  .control-arrow {
+    width: 20px;
+    height: 26px;
+  }
+  /* stylelint-enable */
+}
+
+.image-gallery__slider-control--prev,
+.image-gallery__slider-control--next {
+  background: color('white');
+  width: 48px;
+  height: 48px;
+  line-height: 0;
+  padding: 0;
+  border-radius: 24px;
```

Comment:
There is `icon-rounder`, which might be useful?
There is also `icon--2x`?

https://github.com/wellcometrust/wellcomecollection.org/blob/master/client/scss/base/_icons.scss#L33